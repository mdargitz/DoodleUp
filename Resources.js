import React from 'react';
import {Text} from 'react-native';

export default class Resources extends React.Component {
  render() {
    return (
      <Text>Resources</Text>
    );
  }
}