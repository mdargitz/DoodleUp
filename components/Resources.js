import React from 'react';
import { Text, ImageBackground, View, ScrollView } from 'react-native';
import ResourceCard from './ResourceCard';

export default class Resources extends React.Component {
  render() {
    return (
      <ImageBackground
        source={{ uri: 'https://image.ibb.co/cJfEwd/mesh_1430107_640.png' }}
        style={{ height: '100%', width: '100%' }}
      >
        <View style={styles.viewStyle}>
          <Text style={styles.headerText}> Need to brush up on some skills?</Text>
          <Text style={styles.subtitleText}>Take a look at our currated list of tutorials!</Text>
        </View>
        <ScrollView>
          <Text style={styles.sectionTitle}>Gesture Drawing</Text>
          <ResourceCard
            title='Introduction to Gessture Drawing'
            url='https://www.sophia.org/tutorials/introduction-to-gesture-drawing'
            image='https://sophialearning.s3.amazonaws.com/packet_logos/202783/large/Quick_gesture_drawings.jpg'
          />
          <ResourceCard
            title='Beginner Gesture Drawing'
            url='https://www.youtube.com/watch?v=qdtw6JCQxx8'
            image='http://th09.deviantart.net/fs70/PRE/i/2010/125/0/b/Gesture_Drawing_Studies_by_studionap.jpg'
          />
          <Text style={styles.sectionTitle}> Drawing from Life</Text>
          <ResourceCard
            title='How to Draw What You See'
            url='http://emptyeasel.com/2006/12/08/how-to-draw-what-you-see-techniques-and-tips-to-improve-your-drawing-skills/'
            image='http://emptyeasel.com/wp-content/uploads/2013/08/sketch9-full-range-of-values-large-1.jpg'
          />
          <ResourceCard
            title='Still Life with Pencil'
            url='http://www.artyfactory.com/still-life/still_life_pencil.html'
            image='http://www.artyfactory.com/still-life/images/still_life_pencil_8.jpg'
          />
          <Text style={styles.sectionTitle}>Artistic Concepts</Text>
          <ResourceCard
            title='How to Draw What You See'
            url='http://emptyeasel.com/2006/12/08/how-to-draw-what-you-see-techniques-and-tips-to-improve-your-drawing-skills/'
            image='https://cms-assets.tutsplus.com/uploads/users/108/posts/24486/final_image/think-with-forms-final.png'            
          />
          <ResourceCard
            title='The Beginning Artists Guide to Perspective Drawing'
            url='https://www.artistsnetwork.com/art-mediums/drawing/learn-to-draw-perspective/'
            image='https://www.artistsnetwork.com/wp-content/uploads/2017/06/OnePointPerspectiveReference.png'
          />
          <ResourceCard
            title='A Theory of Light and Shade'
            url='http://www.artinstructionblog.com/drawing-lesson-a-theory-of-light-and-shade'
            image='http://www.artinstructionblog.com/wp-content/uploads/2009/08/values-sheridoty-2009_img_4-300x184.jpg'
          />
          <ResourceCard
            title='The Human Face'
            url='https://design.tutsplus.com/tutorials/human-anatomy-fundamentals-basics-of-the-face--cms-20417'
            image='https://cms-assets.tutsplus.com/uploads/users/110/posts/20417/image/face-front.jpg'
          />
        </ScrollView>
      </ImageBackground>

    );
  }
}

const styles = {
  viewStyle: {
    backgroundColor: 'rgba(0,0,0,.5)',
  },
  headerText: {
    textAlign: 'center',
    color: '#fff',
    paddingTop: 10,
    fontSize: 30,
    fontWeight: '800'
  },
  subtitleText: {
    textAlign: 'center',
    color: '#fff',
    paddingBottom: 10,
    fontSize: 20
  },
  sectionTitle: {
    color: '#fff',
    backgroundColor: 'rgba(0,0,0,.5)',
    fontSize: 25,
    padding: 10,
    textAlign: 'center'
  }
};