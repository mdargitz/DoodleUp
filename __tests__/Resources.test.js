import React from 'react';
import Resources from '../components/Resources';

import renderer from 'react-test-renderer';

test('renders correctly', () => {
  const tree = renderer.create(<Resources />).toJSON();
  expect(tree).toMatchSnapshot();
});