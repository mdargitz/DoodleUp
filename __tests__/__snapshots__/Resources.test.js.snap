// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "height": "100%",
      "width": "100%",
    }
  }
>
  <Image
    source={
      Object {
        "uri": "https://image.ibb.co/cJfEwd/mesh_1430107_640.png",
      }
    }
    style={
      Array [
        Object {
          "bottom": 0,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        },
        Object {
          "height": "100%",
          "width": "100%",
        },
        undefined,
      ]
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,.5)",
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#fff",
          "fontSize": 30,
          "fontWeight": "800",
          "paddingTop": 10,
          "textAlign": "center",
        }
      }
    >
       Need to brush up on some skills?
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#fff",
          "fontSize": 20,
          "paddingBottom": 10,
          "textAlign": "center",
        }
      }
    >
      Take a look at our currated list of tutorials!
    </Text>
  </View>
  <RCTScrollView>
    <View>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "backgroundColor": "rgba(0,0,0,.5)",
            "color": "#fff",
            "fontSize": 25,
            "padding": 10,
            "textAlign": "center",
          }
        }
      >
        Gesture Drawing
      </Text>
      <View>
        <View
          accessibilityComponentType={undefined}
          accessibilityLabel={undefined}
          accessibilityTraits={undefined}
          accessible={true}
          collapsable={undefined}
          hasTVPreferredFocus={undefined}
          hitSlop={undefined}
          isTVSelectable={true}
          nativeID={undefined}
          onLayout={undefined}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#3B3B58",
              "justifyContent": "center",
              "margin": 20,
              "opacity": 1,
              "padding": 30,
              "shadowColor": "#000",
              "shadowOffset": Object {
                "height": 2,
                "width": 0,
              },
              "shadowOpacity": 0.8,
              "shadowRadius": 2,
            }
          }
          testID={undefined}
          tvParallaxProperties={undefined}
        >
          <Image
            source={
              Object {
                "uri": "https://sophialearning.s3.amazonaws.com/packet_logos/202783/large/Quick_gesture_drawings.jpg",
              }
            }
            style={
              Object {
                "height": 200,
                "width": 200,
              }
            }
          />
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            style={
              Object {
                "color": "#fff",
                "fontSize": 20,
                "padding": 10,
                "textAlign": "center",
              }
            }
          >
            Introduction to Gessture Drawing
          </Text>
        </View>
      </View>
      <View>
        <View
          accessibilityComponentType={undefined}
          accessibilityLabel={undefined}
          accessibilityTraits={undefined}
          accessible={true}
          collapsable={undefined}
          hasTVPreferredFocus={undefined}
          hitSlop={undefined}
          isTVSelectable={true}
          nativeID={undefined}
          onLayout={undefined}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#3B3B58",
              "justifyContent": "center",
              "margin": 20,
              "opacity": 1,
              "padding": 30,
              "shadowColor": "#000",
              "shadowOffset": Object {
                "height": 2,
                "width": 0,
              },
              "shadowOpacity": 0.8,
              "shadowRadius": 2,
            }
          }
          testID={undefined}
          tvParallaxProperties={undefined}
        >
          <Image
            source={
              Object {
                "uri": "http://th09.deviantart.net/fs70/PRE/i/2010/125/0/b/Gesture_Drawing_Studies_by_studionap.jpg",
              }
            }
            style={
              Object {
                "height": 200,
                "width": 200,
              }
            }
          />
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            style={
              Object {
                "color": "#fff",
                "fontSize": 20,
                "padding": 10,
                "textAlign": "center",
              }
            }
          >
            Beginner Gesture Drawing
          </Text>
        </View>
      </View>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "backgroundColor": "rgba(0,0,0,.5)",
            "color": "#fff",
            "fontSize": 25,
            "padding": 10,
            "textAlign": "center",
          }
        }
      >
         Drawing from Life
      </Text>
      <View>
        <View
          accessibilityComponentType={undefined}
          accessibilityLabel={undefined}
          accessibilityTraits={undefined}
          accessible={true}
          collapsable={undefined}
          hasTVPreferredFocus={undefined}
          hitSlop={undefined}
          isTVSelectable={true}
          nativeID={undefined}
          onLayout={undefined}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#3B3B58",
              "justifyContent": "center",
              "margin": 20,
              "opacity": 1,
              "padding": 30,
              "shadowColor": "#000",
              "shadowOffset": Object {
                "height": 2,
                "width": 0,
              },
              "shadowOpacity": 0.8,
              "shadowRadius": 2,
            }
          }
          testID={undefined}
          tvParallaxProperties={undefined}
        >
          <Image
            source={
              Object {
                "uri": "http://emptyeasel.com/wp-content/uploads/2013/08/sketch9-full-range-of-values-large-1.jpg",
              }
            }
            style={
              Object {
                "height": 200,
                "width": 200,
              }
            }
          />
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            style={
              Object {
                "color": "#fff",
                "fontSize": 20,
                "padding": 10,
                "textAlign": "center",
              }
            }
          >
            How to Draw What You See
          </Text>
        </View>
      </View>
      <View>
        <View
          accessibilityComponentType={undefined}
          accessibilityLabel={undefined}
          accessibilityTraits={undefined}
          accessible={true}
          collapsable={undefined}
          hasTVPreferredFocus={undefined}
          hitSlop={undefined}
          isTVSelectable={true}
          nativeID={undefined}
          onLayout={undefined}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#3B3B58",
              "justifyContent": "center",
              "margin": 20,
              "opacity": 1,
              "padding": 30,
              "shadowColor": "#000",
              "shadowOffset": Object {
                "height": 2,
                "width": 0,
              },
              "shadowOpacity": 0.8,
              "shadowRadius": 2,
            }
          }
          testID={undefined}
          tvParallaxProperties={undefined}
        >
          <Image
            source={
              Object {
                "uri": "http://www.artyfactory.com/still-life/images/still_life_pencil_8.jpg",
              }
            }
            style={
              Object {
                "height": 200,
                "width": 200,
              }
            }
          />
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            style={
              Object {
                "color": "#fff",
                "fontSize": 20,
                "padding": 10,
                "textAlign": "center",
              }
            }
          >
            Still Life with Pencil
          </Text>
        </View>
      </View>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "backgroundColor": "rgba(0,0,0,.5)",
            "color": "#fff",
            "fontSize": 25,
            "padding": 10,
            "textAlign": "center",
          }
        }
      >
        Artistic Concepts
      </Text>
      <View>
        <View
          accessibilityComponentType={undefined}
          accessibilityLabel={undefined}
          accessibilityTraits={undefined}
          accessible={true}
          collapsable={undefined}
          hasTVPreferredFocus={undefined}
          hitSlop={undefined}
          isTVSelectable={true}
          nativeID={undefined}
          onLayout={undefined}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#3B3B58",
              "justifyContent": "center",
              "margin": 20,
              "opacity": 1,
              "padding": 30,
              "shadowColor": "#000",
              "shadowOffset": Object {
                "height": 2,
                "width": 0,
              },
              "shadowOpacity": 0.8,
              "shadowRadius": 2,
            }
          }
          testID={undefined}
          tvParallaxProperties={undefined}
        >
          <Image
            source={
              Object {
                "uri": "https://cms-assets.tutsplus.com/uploads/users/108/posts/24486/final_image/think-with-forms-final.png",
              }
            }
            style={
              Object {
                "height": 200,
                "width": 200,
              }
            }
          />
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            style={
              Object {
                "color": "#fff",
                "fontSize": 20,
                "padding": 10,
                "textAlign": "center",
              }
            }
          >
            How to Draw What You See
          </Text>
        </View>
      </View>
      <View>
        <View
          accessibilityComponentType={undefined}
          accessibilityLabel={undefined}
          accessibilityTraits={undefined}
          accessible={true}
          collapsable={undefined}
          hasTVPreferredFocus={undefined}
          hitSlop={undefined}
          isTVSelectable={true}
          nativeID={undefined}
          onLayout={undefined}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#3B3B58",
              "justifyContent": "center",
              "margin": 20,
              "opacity": 1,
              "padding": 30,
              "shadowColor": "#000",
              "shadowOffset": Object {
                "height": 2,
                "width": 0,
              },
              "shadowOpacity": 0.8,
              "shadowRadius": 2,
            }
          }
          testID={undefined}
          tvParallaxProperties={undefined}
        >
          <Image
            source={
              Object {
                "uri": "https://www.artistsnetwork.com/wp-content/uploads/2017/06/OnePointPerspectiveReference.png",
              }
            }
            style={
              Object {
                "height": 200,
                "width": 200,
              }
            }
          />
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            style={
              Object {
                "color": "#fff",
                "fontSize": 20,
                "padding": 10,
                "textAlign": "center",
              }
            }
          >
            The Beginning Artists Guide to Perspective Drawing
          </Text>
        </View>
      </View>
      <View>
        <View
          accessibilityComponentType={undefined}
          accessibilityLabel={undefined}
          accessibilityTraits={undefined}
          accessible={true}
          collapsable={undefined}
          hasTVPreferredFocus={undefined}
          hitSlop={undefined}
          isTVSelectable={true}
          nativeID={undefined}
          onLayout={undefined}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#3B3B58",
              "justifyContent": "center",
              "margin": 20,
              "opacity": 1,
              "padding": 30,
              "shadowColor": "#000",
              "shadowOffset": Object {
                "height": 2,
                "width": 0,
              },
              "shadowOpacity": 0.8,
              "shadowRadius": 2,
            }
          }
          testID={undefined}
          tvParallaxProperties={undefined}
        >
          <Image
            source={
              Object {
                "uri": "http://www.artinstructionblog.com/wp-content/uploads/2009/08/values-sheridoty-2009_img_4-300x184.jpg",
              }
            }
            style={
              Object {
                "height": 200,
                "width": 200,
              }
            }
          />
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            style={
              Object {
                "color": "#fff",
                "fontSize": 20,
                "padding": 10,
                "textAlign": "center",
              }
            }
          >
            A Theory of Light and Shade
          </Text>
        </View>
      </View>
      <View>
        <View
          accessibilityComponentType={undefined}
          accessibilityLabel={undefined}
          accessibilityTraits={undefined}
          accessible={true}
          collapsable={undefined}
          hasTVPreferredFocus={undefined}
          hitSlop={undefined}
          isTVSelectable={true}
          nativeID={undefined}
          onLayout={undefined}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#3B3B58",
              "justifyContent": "center",
              "margin": 20,
              "opacity": 1,
              "padding": 30,
              "shadowColor": "#000",
              "shadowOffset": Object {
                "height": 2,
                "width": 0,
              },
              "shadowOpacity": 0.8,
              "shadowRadius": 2,
            }
          }
          testID={undefined}
          tvParallaxProperties={undefined}
        >
          <Image
            source={
              Object {
                "uri": "https://cms-assets.tutsplus.com/uploads/users/110/posts/20417/image/face-front.jpg",
              }
            }
            style={
              Object {
                "height": 200,
                "width": 200,
              }
            }
          />
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            style={
              Object {
                "color": "#fff",
                "fontSize": 20,
                "padding": 10,
                "textAlign": "center",
              }
            }
          >
            The Human Face
          </Text>
        </View>
      </View>
    </View>
  </RCTScrollView>
</View>
`;
